cmd/habitat/community/consensus/raft/service.go

```diff
@@ -60,15 +65,15 @@ func (cs *ClusterService) CreateCluster(communityID string) error {
 
 	stateMachine := fsm.NewCommunityStateMachine()
 
-	ra, ht, err := setupRaftInstance(communityID, stateMachine, true)
+	ra, _, err := setupRaftInstance(communityID, stateMachine, true, cs.host)
```

Comment:
can we remove the return value if it's not used? it seems like it used to be returning the transport layer struct but we don't need that anymore

Reply:
good catch